x-pack/plugins/canvas/i18n/ui.ts

```diff
@@ -1075,10 +1075,14 @@ export const ViewStrings = {
       }),
     getMaintainAspectDisplayName: () =>
       i18n.translate('xpack.canvas.uis.views.shape.args.maintainAspectTitle', {
-        defaultMessage: 'Fixed ratio',
+        defaultMessage: 'Aspect ratio settings',
       }),
-    getMaintainAspectHelp: () =>
+    getMaintainAspectLabelName: () =>
       i18n.translate('xpack.canvas.uis.views.shape.args.maintainAspectLabel', {
```

Comment:
Do we need to do anything to let i18n teams know that these need to be adjusted?

Reply:
I looked through the docs and it doesn't appear to need anything.